### ISSUE TITLE
java.lang.NoSuchMethodError: org.pac4j.core.exception.HttpAction.unauthorized

### ISSUE THREAD

**Issue by eroux** [pac4j/dropwizard-pac4j-demo#8]
Running the demo and clicking on the basic auth link, I'm getting

```
! java.lang.NoSuchMethodError: org.pac4j.core.exception.HttpAction.unauthorized(Ljava/lang/String;Lorg/pac4j/core/context/WebContext;Ljava/lang/String;Ljava/lang/String;)Lorg/pac4j/core/exception/HttpAction;
! at org.pac4j.http.client.indirect.IndirectBasicAuthClient.retrieveCredentials(IndirectBasicAuthClient.java:64)
```

[full log](https://pastebin.com/Xeab03NK)

**Comment by victornoel** [comment]
Thanks, there was a problem with dependency management and pac4j-core was resolved to 2.0.0 instead of latest SNAPSHOT as the rest :)

**Comment by eroux** [comment]
Thanks a lot! works fine now
